includes/User.php

```diff
@@ -36,7 +36,7 @@
  * Int Serialized record version.
  * @ingroup Constants
  */
-define( 'MW_USER_VERSION', 10 );
+define( 'MW_USER_VERSION', 11 );
```

Comment:
This can/will force reload from DB all user data. Do we need this since there is no change in user internal representation?